queuemngr/queue_manager.go

```diff
@@ -144,14 +102,6 @@ func (qc *QueueManager) Stop() error {
 		return err
 	}
```

Comment:
I think existing code is mistake: all these queues are independent so we should call Stop on each one of them, potentially aggregating result error through `errors.Join(...)`